marklogic-data-hub/src/test/java/com/marklogic/hub/deploy/commands/DeployQueryOptionsCommandTest.java

```diff
@@ -0,0 +1,20 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.impl.EntityManagerImpl;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class DeployQueryOptionsCommandTest extends AbstractHubCoreTest {
```

Comment:
I'm not sure this test wouldn't have already passed. You'd first need to verify that the search options don't exist, which may involve deleting the options if they already exist (which they could, due to a prior test running).